code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -7453,6 +7453,44 @@ var/global/list/bomb_like_items = list(/obj/item/device/transfer_valve, /obj/ite
 	reagents.add_reagent(ZAMSPICYTOXIN, 5)
 	bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup
+	name = "Blether Noodle Soup"
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	icon_state = "blethernoodlesoup_open"
+	trash = /obj/item/weapon/reagent_containers/glass/soupcan
+	food_flags = FOOD_MEAT | FOOD_LIQUID
+	crumb_icon = "dribbles"
+	filling_color = "#FF9700"
+	valid_utensils = UTENSILE_FORK|UTENSILE_SPOON
+	wrapped = 0
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 6)
+	reagents.add_reagent(SACID, 10)
+	reagents.add_reagent(LOCUTOGEN, 5)
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/wrapped
+	name = "Blether Noodle Soup"
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	icon_state = "blethernoodlesoup_closed"
+	wrapped = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/attack_self(mob/user)
+	if(wrapped)
+		Unwrap(user)
+		playsound(user, 'sound/effects/can_open1.ogg', 50, 1)
+	else
+		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/proc/Unwrap(mob/user)
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	food_flags = FOOD_MEAT | FOOD_LIQUID
```

Comment:
both the desc and food flags are already inherited

Reply:
Removed